docs/content/TooltipV2.mdx

```diff
@@ -2,10 +2,18 @@
 componentId: tooltip_v2
 title: Tooltip v2
 status: Draft
+a11yReviewed: true
 source: https://github.com/primer/react/blob/main/src/drafts/Tooltip.tsx
 ---
 
-import data from '../../../packages/react/src/drafts/Tooltip/Tooltip.docs.json'
+import data from '../../packages/react/src/drafts/Tooltip/Tooltip.docs.json'
+import {Tooltip as TooltipV2} from '../../packages/react/src/drafts/Tooltip/Tooltip'
```

Comment:
Help needed: [On the Tooltip v2 docs page](https://primer-7bf4cf074a-13348165.drafts.github.io/TooltipV2), the component in the examples is not Tooltip v2, it is clearly still importing the Tooltip v1 but I don't understand why and how to make it import the Tooltip from `experimental` bundle (Tooltip v2)

Reply:
Hi! 

You'd need to extend the `scope` for the code block to include "next" components when requested: 

source: https://github.com/primer/react/blob/main/docs/src/%40primer/gatsby-theme-doctocat/live-code-scope.js#L26-L27

Example usage: https://github.com/primer/react/blob/main/docs/content/deprecated/ActionList.mdx#L47

Reply:
OMG I totally forgot about adding the `jsx live drafts` stuff! Thanks so much. I extend it to include `experimental` and `next` now.